functions/build.sh

```
@@ -0,0 +1,54 @@
+    echo "Processing $function_name..."
+    
+    # Get the ECR URL for this function
+    repo_url=$(echo "$ECR_REPOSITORY_URLS" | jq -r --arg fn "$function_name" '.[$fn]')
+    
+    # Check if the function directory exists and has a Dockerfile
+    if [ -d "$function_name" ] && [ -f "$function_name/Dockerfile" ]; then
+        echo "Building container for $function_name..."
+        
+        # Build and push the container
+        cd "$function_name"
+        docker buildx build --platform linux/arm64 --provenance=false -t $function_name:latest .
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)